src/platform/plugins/shared/visualizations/public/embeddable/interfaces/has_visualize_config.ts

```diff
@@ -18,7 +18,7 @@ export type HasVisualizeConfig = HasType<'visualization'> & {
 export const apiHasVisualizeConfig = (api: unknown): api is HasVisualizeConfig => {
   return Boolean(
     api &&
-      apiIsOfType(api, 'visualization') &&
+      (api as HasType)?.type === 'visualization' &&
```

Comment:
why this change?

Reply:
avoid using `apiIsOfType` to avoid importing presentation-publishing package into page load. Since `apiIsOfType` is so simple, no reason to use the utility method and import such a huge bundle just for it.